test/quantization/core/test_quantized_module.py

```diff
@@ -537,16 +573,16 @@ def test_conv2d_api(self):
                 in_channels_per_group, input_feature_map_size,
                 out_channels_per_group, groups, kernel_size, stride, padding,
                 pad_mode, dilation, X_scale, X_zero_point, W_scale, W_zero_point,
-                Y_scale, Y_zero_point, use_bias, use_fused, use_channelwise)
+                Y_scale, Y_zero_point, use_bias, post_op, use_channelwise)
 
     @skipIfNoFBGEMM
     def test_conv3d_api(self):
         options = itertools.product(
             [True, False],  # use_bias
-            [True, False],  # use_fused
+            ["relu", "none"],  # post_op
```

Comment:
same here

Reply:
Thanks for the suggestions. Split this into multiple tests.